src/main/java/fact/VersionInformation.java

```diff
@@ -0,0 +1,98 @@
+package fact;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class VersionInformation {
```

Comment:
VersionInformation has no javadoc
Also do you realy want to go forth with the version stuff if it can't find a version at all?
I think breaking down would be a good thing as the version should be readable at all times.
I can't think of a reason why you couldn't have a version and still want to do stuff.

Reply:
The only case where it is not readable is when I run tests directly from intellij, I want to keep this functionality.